### PR TITLE
games-misc/wtf: bump package

### PR DESCRIPTION
**Pull request by waebbl**
Package-Manager: Portage-3.0.14, Repoman-3.0.2
Signed-off-by: Bernd Waibel <waebbl@gmail.com>

### CODE REVIEW

**Review by ConiKost — CHANGES_REQUESTED** [review]
Please split your commit into two. One adds the new version, a second one drops the old version. Don't do that in one commit.

File: games-misc/wtf/wtf-20210130.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+inherit prefix
+
+DESCRIPTION="Translates acronyms for you"
+HOMEPAGE="http://netbsd.org/"
+SRC_URI="https://sourceforge.net/projects/bsd${PN}/files/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~m68k ~mips ~ppc64 ~s390 ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
+IUSE=""
```
Comment (ConiKost) [inline games-misc/wtf/wtf-20210130.ebuild]: If you don't have any use flags, you can drop ```IUSE```.

File: games-misc/wtf/wtf-20210130.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit prefix
+
+DESCRIPTION="Translates acronyms for you"
+HOMEPAGE="http://netbsd.org/"
```
Comment (ConiKost) [inline games-misc/wtf/wtf-20210130.ebuild]: Could check, if ```HOMEPAGE``` is also reachable via ```https``` and update, if so?

**Review by ConiKost — CHANGES_REQUESTED** [review]
Just a few small nit picks :-)

Could you also say in your commit for bumping version, to which version you are bumping?
Thanks!

**Comment by waebbl** [comment]
> Just a few small nit picks :-)
> 
> Could you also say in your commit for bumping version, to which version you are bumping?
> Thanks!

Ah yes, of course. Usually I do this, don't why not here...
I also fix the other two issues.

**Review by ConiKost — APPROVED** [review]

**Comment by ConiKost** [comment]
Commited.
https://github.com/gentoo/gentoo/commit/4474b12134a18bdad61f88f68de85b0343668d16

**Comment by waebbl** [comment]
Thank you @ConiKost

**Pull request closed without merging**